### PR TITLE
Adding pages

### PR DESCRIPTION
**Pull request by ZucchiniZe**
Add title and layout to all pages
Remove Blog
Not my icon, [source](http://dribbble.com/shots/1456970-Atom-io-Icon)

### CODE REVIEW

**Comment by jbranchaud** [comment]
- Your first commit included an icon, what did you have in mind for that?
- I am not sure why you removed the blog itself, I intend to use that feature, though the default content that came with this Jekyll theme does still need to be removed.
- I noticed you modified the `baseurl` value to `""`. See #46 for why that is problematic.
- A couple things need to happen before all those markdown documents can be converted to _pages_ including hyphenating the names (per #44) and figuring out how that content will be organized (so that the sidebar doesn't blow up with 20+ odd links) -- I should have already opened an issue regarding my concerns in this last point. My bad.

Two side notes:
- Make sure to open up issues before making extensive changes like this so that we can confirm it is a change we want and make sure it gets assigned to a single person. That is the best way we can make sure no one's effort goes to waste.
- We should do our best to keep each pull request focused on accomplishing/fixing a single thing. You have tried to tackle a couple things above making your changes _all or nothing_.

tl;dr I cannot merge this PR as is, but if we open up some new issues, we can figure out how to move forward with each.

**Comment by ZucchiniZe** [comment]
Sorry, I'm a bit new to this whole github thing.

**Comment by ZucchiniZe** [comment]
With the icon I was thinking of using it like [so](http://i.imgur.com/zqUSxND.png)

**Comment by jbranchaud** [comment]
> Sorry, I'm a bit new to this whole github thing.

No worries. I feel like there is a lot of etiquette and best practices that aren't documented anywhere and so you just kinda have to pick it up as you go. Plus every project is a little different (which I suppose is what `CONTRIBUTING.md` should be used for).

> With the icon I was thinking of using it like so

I am hoping we can come up with an icon/logo and use it like you are suggesting, but I am not sure that is the right icon/logo. Also, it is not clear (to me) how the creator has licensed that icon. That is, I am not sure we can freely plug it into the site anyway.
